features/steps/steps.go

```diff
@@ -241,3 +250,27 @@ func (c *IdentityComponent) theResponseShouldMatchTheFollowingJsonForListgroups(
 	}
 	return nil
 }
+
+func (c *IdentityComponent) theResponseShouldMatchTheFollowingJsonForListgroupsusers(body *godog.DocString) (err error) {
```

Comment:
There is a step in `dp-component-test` you could use for this:

`I should receive the following JSON response:`

https://github.com/ONSdigital/dp-component-test/blob/6f7c3b28d792614796490189bacc3458f5db487d/api_feature.go#L64

Reply:
it's more generic so more re-usable.